src/task.ts

```diff
@@ -9,6 +9,22 @@ export type Quest<T> = {
   tasks: T[];
 };
 
+export type Modes = {
+  backupcamera?: "ml" | "meat" | "init";
+  umbrella?: "broken" | "forward" | "bucket" | "pitchfork" | "twirling" | "cocoon";
+  snowsuit?: "eyebrows" | "smirk" | "nose" | "goatee" | "hat";
+  edpiece?: "bear" | "owl" | "puma" | "hyena" | "mouse" | "weasel" | "fish";
+  retrocape?: ["vampire" | "heck" | "robot", "hold" | "thrill" | "kiss" | "kill"];
```

Comment:
Phred has said he didn't like how this tuple worked out in practice, since sometimes you only care about the superhero. Can we allow "vampire" | "heck" | "robot" as possible values as well?